lib/plugins/pluginUtils.js

```diff
@@ -63,12 +63,12 @@ function extractAttributesArrayFromNgsi2Entity(entity) {
  * @param      {Object}  attsArray  The atts array
```

Comment:
Function documentation must be updated with the new parameter

Reply:
fixed in a79ffd8438cadf1fd25a5cc22e3e49ec8170a437